views/make/index.js

```diff
@@ -35,6 +35,10 @@ module.exports = view.extend({
         goBack: function (e) {
             e.preventDefault();
             if (this.$data.mode === 'settings') {
+                app.update({
```

Comment:
this will actually add new properties to the remote data model, which is not what we want. i think it might be better to save the old values and re-apply them on cancel  instead, or else create some stand-alone properties on `data` and use those for the settings view instead. what do you think?